atomic_reactor/util.py

```diff
@@ -1591,33 +1591,47 @@ def get_platform_config(platform, build_annotations):
 def annotation(*keys):
     """
     Annotate a `BuildPlugin` subclass. The `run()` method of this plugin will
-    store its results in the plugin's workflow as annotations.
+    store its result in the plugin's workflow as annotations.
 
-    The `run()` method of the plugin being annotated has to return a dict
-    containing all of the specified keys or None (no annotations will be set).
+    If run() returns None, no annotations will be set.
 
     The `store_metadata_in_osv3` plugin will automatically collect these
     annotations and upload them to OpenShift.
 
     Example:
-    >>> from atomic_reactor.plugin import BuildPlugin
-
     >>> @annotation('foo', 'bar')
     >>> class MyBuildPlugin(BuildPlugin):
     >>>     key = 'my_build_plugin'
     >>>
+    >>>     # sets annotations: {'foo': 1, 'bar': 1}
```

Comment:
Not sure about this. It sets the same value on both annotations? It doesn't seem like useful behaviour to duplicate it.

Should it just take a single argument (the annotation name)? If it takes multiple arguments I'd expect it to unpack a tuple from the return value.

Reply:
Fair point, I don't think I've seen any plugins returning tuples so I will probably just make it take only a single argument.